### PR TITLE
Created FlipCard component and Teams component

### PR DESCRIPTION
**Pull request by mvkv**

### CODE REVIEW

File: src/components/Teams/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        name: "Nome cognome",
+        image: Img,
+        description: "Lorem ipsum dolor sit amet...",
+      },
+    ],
+  },
+];
+
+export const Primary = () => <Teams teams={teams} />;
+export { default } from "./Teams";
```
Comment (lynxnb) [inline src/components/Teams/index.js:66]: Shouldn't this file only have this line? It looks like it's a duplicate of `Teams.stories.js`

File: src/components/Teams/Teams.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            teamComponents: [
+                {
+                    name,
+                    image,
+                    description,
+                },
+            ]
+        },
+    ]
+  */
```
Comment (lynxnb) [inline src/components/Teams/Teams.js:29]: Offtopic: this is why I have JavaScript. Weak typing is so bad 😭

**Review by lynxnb — APPROVED** [review]
Looks good apart from that `index.js` file, I'll approve anyways as I'll know you'll fix it

File: src/components/Teams/index.js (hunk 1 above)
Comment (mvkv) [reply src/components/Teams/index.js:66]: Fixed

**Pull request merged**